addons/osfstorage/tests/test_views.py

```diff
@@ -373,7 +373,7 @@ def test_upload_duplicate(self):
 
     def test_upload_create_child(self):
         name = 'ლ(ಠ益ಠლ).unicode'
-        parent = self.node_settings.get_root().append_folder('cheesey')
+        parent = self.node_settings.get_root().append_folder('cheesy')
```

Comment:
I'm assuming no one cares.

Note that I don't object to cute terms. I just prefer canonical cute terms (I made an exception for Freddie Mercury as I couldn't tell what the code was doing)